### PR TITLE
[PE-4854] Backstop test for Badge Component

### PR DESCRIPTION
**Pull request by sabah1910**
@jllr @janibolkvadze @mattnickles https://ctl.atlassian.net/browse/FSPE-4854

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://e7a5b98f1e51a5954b8c2c877a21403184ef5702-chi-pr.enterprise.ctl.io" target="_blank"> https://e7a5b98f1e51a5954b8c2c877a21403184ef5702-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1098/ for more details.

File: test/chi/components/badges.pug [hunk 1]
```diff
@@ -2,31 +2,41 @@
...
-        }
-        )
-          .chi-badge__content
-            .chi-icon
-              svg
-                use(xlink:href='#icon-atom')
-            span= style
+    div(class={
+      [`test-${type}`]: size === 'default',
+      [`test-${type}-${size}`]: size !== 'default'
```
Comment (jllr) [inline test/chi/components/badges.pug:9]:
```suggestion
    div(class=`test-${type}`)
```

File: test/chi/components/badges.pug [hunk 2]
```diff
@@ -54,3 +64,14 @@ h2 Badge Icons
...
+
+h2 Text Transform
+.test-text-transform
+  .example.-mb--1
+        .chi-badge.-text--uppercase
+          span Badge
+        .chi-badge.-text--lowercase
+          span Badge
+        .chi-badge.-text--capitalized
+          span Badge
```
Comment (jllr) [inline test/chi/components/badges.pug:76]:
```suggestion
       .p.-text--bold 
          | Uppercase
          .chi-badge.-text--uppercase
            span Badge
        .p.-text--bold 
          | Lowercase
          .chi-badge.-text--lowercase
          span Badge
        .p.-text--bold 
          | Capitalized
          .chi-badge.-text--capitalized
          span Badge
```

**Comment by jllr** [comment]
@sabah1910 after these changes are done, be sure indentation is ok and run the tests again. Before sending the PR make sure that the generated images reference corresponds to the changed tests.

**Comment by sabah1910** [comment]
@jllr  Thanks, Jose. Will make sure to do so.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://83b60bc81727fa7754aa1331ae53908bbde45b46-chi-pr.enterprise.ctl.io" target="_blank"> https://83b60bc81727fa7754aa1331ae53908bbde45b46-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1101/ for more details.

**Comment by jllr** [comment]
test this please

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://83b60bc81727fa7754aa1331ae53908bbde45b46-chi-pr.enterprise.ctl.io" target="_blank"> https://83b60bc81727fa7754aa1331ae53908bbde45b46-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1102/ for more details.

**Comment by janibolkvadze** [comment]
@sabah1910 Build failed because you also need to update tests of Badge component custom elements. Please check out: `test/chi/custom-elements/badge.pug` file

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://7a2e0f260ef07cc76ff4e2ba19798f6858004990-chi-pr.enterprise.ctl.io" target="_blank"> https://7a2e0f260ef07cc76ff4e2ba19798f6858004990-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1118/ for more details.

**Comment by sabah1910** [comment]
@jllr @janibolkvadze @mattnickles Updated custom elements for badge, but build still failed. Trying to figure out why.

**Comment by jllr** [comment]
test this please

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://7a2e0f260ef07cc76ff4e2ba19798f6858004990-chi-pr.enterprise.ctl.io" target="_blank"> https://7a2e0f260ef07cc76ff4e2ba19798f6858004990-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1130/ for more details.

**Comment by jllr** [comment]
@sabah1910 before sending a PR, make sure that you test it locally with chi test (visual tests) and chi test-e2e (functional tests). Both tests are failing right now and this is the reason why the PR is not building in Jenkins.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://4a9f5d90d0734da0953b4ab9efc04e53bd636cec-chi-pr.enterprise.ctl.io" target="_blank"> https://4a9f5d90d0734da0953b4ab9efc04e53bd636cec-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1192/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://04c7d864d546330d5dcc198fdbaa0e57c6a43f39-chi-pr.enterprise.ctl.io" target="_blank"> https://04c7d864d546330d5dcc198fdbaa0e57c6a43f39-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1200/ for more details.

**Comment by sabah1910** [comment]
@jllr @mattnickles @janibolkvadze https://ctl.atlassian.net/browse/FSPE-4854

File: src/custom-elements/src/components/date/date.tsx [hunk 3]
```diff
@@ -150,7 +150,7 @@ export class Date {
   };
 
   private _initCalendarViewModel(): void {
-    dayjs.locale(this.locale);
+    // dayjs.locale(this.locale);
```
Comment (janibolkvadze) [inline src/custom-elements/src/components/date/date.tsx:153]: @sabah1910 Please uncomment this line and Rebase from master

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://c346090774fb01e14911bd04099616217e2c4f7e-chi-pr.enterprise.ctl.io" target="_blank"> https://c346090774fb01e14911bd04099616217e2c4f7e-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1223/ for more details.

File: src/custom-elements/src/components/date/date.tsx (hunk 3 above)
Comment (sabah1910) [reply src/custom-elements/src/components/date/date.tsx]: @janibolkvadze Done

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-ewzddj, https://ops.elasticbox.com/services/instances/i-atahku, https://ops.elasticbox.com/services/instances/i-4mkt2y, https://ops.elasticbox.com/services/instances/i-cw98qu

**Pull request merged**